### DISCUSSION TITLE
CoApp dilemma

### DISCUSSION THREAD

**Discussion by blondie007** [aclap-dev/video-downloadhelper#415] [Help and Q&A]
### Your question

I've just bought a new Mac Studio (Apple M2 Ultra chip). OS Sonoma v14.4.1
I need to install the CoApp but when I followed the link, I'm offered 2 choices - Intel or Apple silicon.
How should I know which to choose?

### Addon & CoApp version and details

Video DownloadHelper
Version 8.2.2.8 (stable)
Browser locale: en-US
Production build
Built on 2024-03-18
Build options: browser=firefox, linuxlic=false, noyt=false, target=mozilla

**Comment by jcfrog** [comment]
Hi,
M2 are "Apple Silicon": https://www.apple.com/newsroom/2023/01/apple-unveils-m2-pro-and-m2-max-next-generation-chips-for-next-level-workflows/

**Reply by blondie007** [reply]
Thanks a bunch, jcfog! It's all gobbledigook to me. I'll now install the coapp on the new beastie, and hopefully it will capture vids from our abc & sbs tv.

**Reply by Mattmjs** [reply]
@blondie007 , do mean Australian tv channels ?

**Comment by blondie007** [comment]
Yes, that's correct.
________________________________

**Reply by Mattmjs** [reply]
So abc has all their content protected with DRM unfortunately, if you've tried downloading any of their videos you end
up with a scrambled encrypted file.

**Comment by blondie007** [comment]
That’s a shame; vdh used to capture the vids beautifully. A friend of mine with a PC is able to d/l the SBS vids using a windows app, so it looks like I’ll need to invest in a PC as well. What a pain in the hip pocket. :(  Thanks for replying.

**Reply by Mattmjs** [reply]
You might be ok to use vdh with SBS, I'm not aware of them protecting the video content. It's just won't work on abc.

**Comment by blondie007** [comment]
Sadly no. I just tried and all I got was a "MPEG-4 (Base Media): 30.8 MiB" file. So with great regret I can no longer support your app.
________________________________
